### ISSUE TITLE
Can't install biosig (make: *** No rule to make target `install_save2gdf`)

### ISSUE THREAD

**Issue by nbara** [schloegl/homebrew-biosig#7]
Hi there,

I can't install biosig on my machine. I get the following error :

```
→ brew install biosig
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 2 taps (caskroom/cask, homebrew/core).
==> Updated Formulae
httpie           icdiff           kobalt           lean-cli         mysql-sandbox    youtube-dl

==> Installing biosig from schloegl/biosig
==> Downloading http://pub.ist.ac.at/~schloegl/biosig/prereleases/biosig4c++-1.8.4c.src.tar.gz
Already downloaded: /Users/nicolas/Library/Caches/Homebrew/biosig-1.8.4.tar.gz
==> PKG_CONFIG_PATH=/usr/local/lib/pkgconfig make install_save2gdf
Last 15 lines from /Users/nicolas/Library/Logs/Homebrew/biosig/01.pkgconfig:
2017-04-17 11:04:38 +0200

PKG_CONFIG_PATH=/usr/local/lib/pkgconfig make install_save2gdf

make: *** No rule to make target `install_save2gdf'.  Stop.

```

Do you have any suggestions ?

<details> 
<summary>`brew --config` output </summary>

```
HOMEBREW_VERSION: 1.1.13
ORIGIN: https://github.com/Homebrew/brew
HEAD: 46f6dc4c93b96abd528c3a0a808a5ca5a795c7f3
Last commit: 25 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: e320d1de5f1ec8bf4f9b0f9be7455e20ed323384
Core tap last commit: 83 minutes ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local/Homebrew
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: octa-core 64-bit haswell
Homebrew Ruby: 2.0.0-p648
Clang: 8.1 build 802
Git: 2.12.2 => /usr/local/bin/git
Perl: /usr/bin/perl
Python: /Users/nicolas/anaconda/bin/python => /Users/nicolas/anaconda/bin/python2.7
Ruby: /usr/bin/ruby => /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
Java: 1.6.0_65-b14-468
macOS: 10.12.4-x86_64
Xcode: 8.3.1
CLT: N/A
X11: 2.7.9 => /opt/X11
```

</details> 

<details>
<summary>`brew doctor` output </summary>

```
Warning: Anaconda is known to frequently break Homebrew builds, including Vim and
MacVim, due to bundling many duplicates of system and Homebrew-available
tools.

If you encounter a build failure please temporarily remove Anaconda
from your $PATH and attempt the build again prior to reporting the
failure to us. Thanks!

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and what additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew provided
script of the same name. We found the following "config" scripts:
  /Users/nicolas/anaconda/bin/curl-config
  /Users/nicolas/anaconda/bin/freetype-config
  /Users/nicolas/anaconda/bin/icu-config
  /Users/nicolas/anaconda/bin/libdynd-config
  /Users/nicolas/anaconda/bin/libpng-config
  /Users/nicolas/anaconda/bin/libpng16-config
  /Users/nicolas/anaconda/bin/python-config
  /Users/nicolas/anaconda/bin/python2-config
  /Users/nicolas/anaconda/bin/python2.7-config
  /Users/nicolas/anaconda/bin/xml2-config
  /Users/nicolas/anaconda/bin/xslt-config

Warning: Python is installed at /Library/Frameworks/Python.framework

Homebrew only supports building against the System-provided Python or a
brewed Python. In particular, Pythons installed to /Library can interfere
with other software installs.

Warning: Unbrewed dylibs were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected dylibs:
  /usr/local/lib/libblkid.2.0.dylib
  /usr/local/lib/libcom_err.1.1.dylib
  /usr/local/lib/libe2p.2.1.dylib
  /usr/local/lib/libext2fs.2.1.dylib
  /usr/local/lib/liblo.7.dylib
  /usr/local/lib/libss.1.0.dylib
  /usr/local/lib/libtcl8.6.dylib
  /usr/local/lib/libtk8.6.dylib
  /usr/local/lib/libuuid.1.1.dylib

Warning: Unbrewed header files were found in /usr/local/include.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected header files:
  /usr/local/include/tcl.h
  /usr/local/include/tclDecls.h
  /usr/local/include/tclOO.h

Warning: Unbrewed .la files were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected .la files:
  /usr/local/lib/libltdl.la

Warning: Unbrewed .pc files were found in /usr/local/lib/pkgconfig.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected .pc files:
  /usr/local/lib/pkgconfig/tcl.pc
  /usr/local/lib/pkgconfig/tk.pc

Warning: Unbrewed static libraries were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected static libraries:
  /usr/local/lib/libtclstub8.6.a
  /usr/local/lib/libtkstub8.6.a

Warning: You have a curlrc file
If you have trouble downloading packages with Homebrew, then maybe this
is the problem? If the following command doesn't work, then try removing
your curlrc:
  curl https://github.com

Warning: Homebrew's sbin was not found in your PATH but you have installed
formulae that put executables in /usr/local/sbin.
Consider setting the PATH for example like so
  echo 'export PATH="/usr/local/sbin:$PATH"' >> ~/.zshrc
```

</details>  

(note that I tried installing without anaconda in the PATH, but got the same error)

**Comment by schloegl** [comment]
Thanks for your interest. As a work-around, I suggest to install it from
source,


|   git clone git://git.code.sf.net/p/biosig/code biosig-code |
      cd  biosig-code/biosig4c++
      make
      make tools
      sudo make install


should do what you want. Patches are welcome.



Best,
  Alois

**Comment by nbara** [comment]
Hi @schloegl, unfortunately I can't compile biosig from source on my mac (that's why I was looking forward to use homebrew-biosig)

```
gawk -f eventcodes.awk "extern/eventcodes.txt"
awk -f annotatedECG.awk "extern/11073-10102-AnnexB.txt" > "11073-10102-AnnexB.i"
cc -c  -D=NO_OLDNAMES -D=WITH_ZLIB -D=__4HAERTEL__ -D=WITH_FAMOS -D=WITH_FIFF -D=WITH_CHOLMOD -D=WITHOUT_NETWORK -D=WITH_FEF -D=WITH_ATF -D=MAKE_EDFLIB -std=gnu11 -pipe -fPIC -fno-builtin-memcmp -O2 -Wno-unused-result -Wno-deprecated -D_GNU_SOURCE -D_XOPEN_SOURCE=700 -D_DEFAULT_SOURCE -I/usr/local/include -mmacosx-version-min=10.7   -o "biosig.o" "biosig.c"
In file included from biosig.c:63:
./biosig-dev.h:73:18: fatal error: 'cholmod.h' file not found
        #include <cholmod.h>
                 ^
1 error generated.
make: *** [biosig.o] Error 1
```

**Comment by cbrnr** [comment]
I can confirm this problem. If you only need the library, `brew install libbiosig` works.

Regarding the compilation error, it looks like `/usr/local/include` is not in your include path. It is likely that you only have Xcode installed but not the command line tools (CLT). The easiest way to fix this is to install the CLT via `xcode-select --install` (see [here](https://github.com/Homebrew/brew/issues/556) for an explanation why this works).

Regarding the error when installing the formula `biosig`, the rule for the specified target is missing. @schloegl is the target `install_save2gdf` needed? If so, could you add the missing rule?

**Comment by schloegl** [comment]
This issue should be solved with the latest upgrade to libbiosig v1.9.0
